app/api/profiles/route.ts

```diff
@@ -0,0 +1,21 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import {
+  allRequestController,
+  getRequestController,
+} from '../../../backend/utills/http.utills'
+
+export default async function routeHandler(
+  req: NextApiRequest,
+  res: NextApiResponse,
+) {
+  await allRequestController(req, async () => {
```

Comment:
**issue:** This function `allRequestController` currently makes no sense, as you're duplicating `switch` for `req.method`. I will leave you and example of how can we fix it in next comment